docs/man/man1/oc-ex-dockerbuild.1

```diff
@@ -17,7 +17,12 @@ Build a Dockerfile into a single layer
 
 .PP
 Builds the provided directory with a Dockerfile into a single layered image.
-Requires that you have a working connection to a Docker engine.
+Requires that you have a working connection to a Docker engine. You may mount
```

Comment:
Just so we have a vehicle for it to be testable.  Eventually I think both this and s2i commands would be under the CLI.